setup.cfg

```diff
@@ -1,4 +1,4 @@
 [flake8]
-ignore = F403
+ignore = F403, F405
```

Comment:
What's the need for adding this?

Reply:
Due to import * is used some files. Exactly not remember which files.

Reply:
I don't see any star imports though